tests/data/a.html

```diff
@@ -1 +1 @@
-<a href="https://eff.org/">Electronic Frontier Foundation</a>
+<a href="https://eff.org/">Electronic Frontier Foundation</a>
```

Comment:
Please avoid unrelated whitespace changes in the test input. Existing test files should remain as is. (multiple times)